docs/Fibonacci.md

```diff
@@ -0,0 +1,17 @@
+# Fibonacci
+
+ - `v::arrayType()`
+
+Validates whether the input follows the fibonacci integer sequence
```

Comment:
fibonacci must always starts with F in upper case, also you forget the dot (.) in the end of this paragraph.